pandas/io/tests/test_gbq.py

```diff
@@ -1124,7 +1228,7 @@ def setUpClass(cls):
         # executing *ALL* tests described below.
 
         _skip_if_no_project_id()
-        _skip_if_no_private_key_path()
+        _skip_local_auth_if_in_travis_env()
 
         _setup_common()
         clean_gbq_environment(_get_private_key_path())
```

Comment:
Since we are testing local user account auth, can we change this to `clean_gbq_environment()`? We should also make the same change in tearDownClass.